fido/fido.py

```diff
@@ -151,9 +217,11 @@ def get_agent(reactor, connect_timeout=None):
     return ProxyAgent(http_proxy_endpoint)
 
 
-def fetch(url, timeout=None, connect_timeout=None, method='GET',
-          content_type=DEFAULT_CONTENT_TYPE, user_agent=DEFAULT_USER_AGENT,
-          headers=None, body=''):
+def fetch(
+    url, method='GET', headers=None, body='', timeout=DEFAULT_TIMEOUT,
+    connect_timeout=DEFAULT_CONNECT_TIMEOUT,
+    content_type=DEFAULT_CONTENT_TYPE, user_agent=DEFAULT_USER_AGENT,
+):
     """Make an HTTP request.
 
     :param url: the URL to fetch.
```

Comment:
I can't comment on it, but :param timeout: docstring is inaccurate. There is a default now. Want to update the docstring?